<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teaAddSco</class>
 <widget class="QDialog" name="teaAddSco">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>178</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>添加学生成绩</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="Label">
       <property name="text">
        <string>课程名称</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="classEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="Label_2">
       <property name="text">
        <string>学生学号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="stuIdEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="Label_3">
       <property name="text">
        <string>学生名称</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="stuNameEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="Label_4">
       <property name="text">
        <string>学生分数</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="ScoEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Label_5">
       <property name="text">
        <string>学生班级</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lessonEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="Label_6">
       <property name="text">
        <string>学生学院</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="collegeEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="Label_7">
       <property name="text">
        <string>学生专业</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="majorEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="checkBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
